src/main/java/seedu/address/model/ModBook.java

```diff
@@ -110,6 +110,24 @@ public void removeLesson(Module module, Lesson target) {
         module.getLessons().remove(target);
     }
 
+    /**
+     * Sets Exam {@code target} in the Module {@code module}.
+     * {@code target} must exist in {@code module}'s exams list.
+     */
+    public void setExam(Module module, Exam target, Exam newExam) {
+        List<Exam> examList = module.getExams();
```

Comment:
maybe to satisfy law of demeter (i.e. don't talk to strangers), can implement `Module::setExam` and `Module::setLesson` methods?

if you do this then you'll probably also have to go and edit the delete/add lesson/exam methods in `ModelManager`, shown below:
![image](https://user-images.githubusercontent.com/68138671/137845402-9df7bd1c-e183-46e4-b1f6-afaeda7447a2.png)
like right now there are 2 different ways of doing similar things, and yours looks more like the remove methods. should standardise la.